### PR TITLE
add payload to logs for debuging

### PR DESCRIPTION
**Pull request by mrmakeit**
as per #46.

### CODE REVIEW

File: module.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ var hook = function(req, res) {
 
 var github = function(payload, localToken) {
     // TODO: Only acknowledge pushes to the "Master" branch.
-    console.log("Repo: " + payload.repository.full_name);
+    console.log('Repo: ' + payload.repository.full_name);
+    console.log('Payload: ');
+    console.log(payload);
```
Comment (openjck) [inline module.js]: Should we combine these like `console.log('Payload: ' + payload)`?

File: module.js [hunk 2]
```diff
@@ -16,7 +16,9 @@ var hook = function(req, res) {
 
 var github = function(payload, localToken) {
     // TODO: Only acknowledge pushes to the "Master" branch.
-    console.log("Repo: " + payload.repository.full_name);
+    console.log('Repo: ' + payload.repository.full_name);
+    console.log('Payload: ');
```
Comment (openjck) [inline module.js]: Looks like we don't need a trailing space, since the `:` will be the last readable character printed on that line.

**Comment by openjck** [comment]
Needs a rebase

**Comment by mrmakeit** [comment]
@groovecoder maybe it's time to add CleanPR?

**Comment by groovecoder** [comment]
giphy

**Comment by openjck** [comment]
Needed another rebase. I took care of it and pushed it to master.

https://github.com/mdn/discord/commit/28247237575bd8bb037c3ea514d9310467683b0c

**Pull request closed without merging**